azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java

```diff
@@ -331,19 +322,20 @@ public void run() {
                   "Trigger manager thread " + this.getName() + " is too busy! Remaining idle time in ms: "
                       + TriggerManager.this.runnerThreadIdleTime);
             } else {
-              TriggerManager.this.syncObj.wait(TriggerManager.this.runnerThreadIdleTime);
+              Thread.sleep(TriggerManager.this.runnerThreadIdleTime);
```

Comment:
Can you explain a little about the old dynamic of `this.syncObj` and why it's safe to directly sleep the current Thread instead?

Reply:
Again it's a good habit to check java doc directly. An obj's wait method is a direct waiting to be invoked operation, anything interruption would awake this obj to continue with the code. -> of course in the original implementation, there are keywords "synchrinozed", which means others need to wait when someone ontain this obj's lock. But I remove them in this PR, when this syncObj only remains the "sleep" functionality.